analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
are the plots for inter- vs. intra- interaction similarity accurate?

Reply:
also MSE

Reply:
Can't change anything about them. I think the score function just gives bad results